recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "rubin-env" %}
-{% set version = "0.3.1" %}
+{% set version = "0.3.2" %}
```

Comment:
@ktlim did we decide that adding a new package should be a minor version increment?

Reply:
Yes, we did.  This should be 0.4.0.

Reply:
That was just a wild guess on my part.  Fixed.